src/index.js

```diff
@@ -0,0 +1,18 @@
+import './style.css';
```

Comment:
- Kindly work on the requested [design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).
![Screen Shot 2021-08-13 at 21 03 02](https://user-images.githubusercontent.com/22258274/129431209-56d01e6b-176c-4bde-b2f7-c19b913a3f18.png)